### PR TITLE
NXP-29878: fix platform target version

### PR DESCRIPTION
**Pull request by doulba**

### CODE REVIEW

**Review by andreacornaglia — APPROVED** [review]

**Comment by RSalem07** [comment]
@doulba the build is fail, https://jenkins.napps.dev.nuxeo.com/job/nuxeo/job/nuxeo-retention/view/change-requests/job/PR-28/ perhaps some ressource issues, anyway I restared it https://jenkins.napps.dev.nuxeo.com/job/nuxeo/job/nuxeo-retention/view/change-requests/job/PR-28/

**Comment by doulba** [comment]
> @doulba the build is fail, https://jenkins.napps.dev.nuxeo.com/job/nuxeo/job/nuxeo-retention/view/change-requests/job/PR-28/ perhaps some ressource issues, anyway I restared it https://jenkins.napps.dev.nuxeo.com/job/nuxeo/job/nuxeo-retention/view/change-requests/job/PR-28/

Hello @RSalem07 
indeed we have some issues with the resources

File: ci/jenkinsfiles/common-lib.groovy [hunk 1]
```diff
@@ -121,8 +121,8 @@ void deployPreview(String namespace, String charDir, String isCleanupPreview, St
         ${traceCmd}
         ${previewCmd}
       """
-    } catch (err){
-      if (isCleanupPreview == 'true'){
+    } catch (err) {
+      if (isCleanupPreview == 'true') {
         this.cleanupPreview("${namespace}")
```
Comment (RSalem07) [inline ci/jenkinsfiles/common-lib.groovy:126]: is this a cleaning :)

File: ci/helm/preview/values.yaml [hunk 2]
```diff
@@ -85,24 +85,25 @@ nuxeo:
...
-    tolerations:
-    - key: team
-      operator: Equal
-      value: napps
-      effect: NoSchedule
-    nodeSelector:
-      team: napps
+    master:
+      labels:
+        branch: "$BRANCH_NAME"
```
Comment (RSalem07) [inline ci/helm/preview/values.yaml:97]: I don't know if we have a best practice on these conf, I mean should we put at the beginning of the file itself

**Review by RSalem07 — COMMENTED** [review]
👍 Left two questions / comments

File: ci/jenkinsfiles/common-lib.groovy (hunk 1 above)
Comment (doulba) [reply ci/jenkinsfiles/common-lib.groovy:126]: Yes it is

File: ci/helm/preview/values.yaml (hunk 2 above)
Comment (doulba) [reply ci/helm/preview/values.yaml:97]: We can follow what did the platform team

**Review by RSalem07 — APPROVED** [review]

**Pull request merged**